heisenbridge/__main__.py

```diff
@@ -572,6 +595,13 @@ async def run(self, listen_address, listen_port, homeserver_url, owner, safe_mod
             try:
                 config = await self.az.intent.get_account_data("irc", room_id)
 
+                if config.get("is_hidden_room", False):
+                    # Migrate standalone hidden room
+                    config["type"] = HiddenRoom.__name__
+                    config["user_id"] = None
+                    del config["is_hidden_room"]
+                    await self.az.intent.set_account_data("irc", config, room_id)
+
```

Comment:
This section will be dropped before merging, it only exists to migrate my own server.